index.html

```diff
@@ -143,7 +143,19 @@ <h3 class="social-media">LET’S CONNECT</h3>
         </ul>
       </section>
     </section>
-    
+    <section class="form-section">
+      <h2>Contact me</h2>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      <form action="https://formspree.io/f/xzbqzgww" class="info-form" method="post">
+        <input type="text" placeholder="Yeremias" required class="input-text" maxlength="30">
+        <input type="email" placeholder="natajayanj14@gmail.com" class="input-text" maxlength="30" required>
+        <textarea name="form-description" id="form-text-area" cols="30" rows="8" maxlength="500" placeholder="Write your message here"></textarea>
```

Comment:
While testing your validations, I noticed that the form submits even when this field is left empty, so I suggest that you add some extra validations here using the `required` attribute. This will ensure that potential contacts do not submit the form without a proper message describing why they are contacting you.